packages/mgt-components/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.ts

```diff
@@ -617,8 +627,11 @@ export class MgtTeamsChannelPicker extends MgtTemplatedComponent {
               return html`
                 <fluent-tree-item
                   ?expanded=${obj?.isExpanded}
-                  @click=${(e: Event) => this.handleTeamTreeItemClick(e)}>
-                    ${icon}${obj.item.displayName}
+                  aria-live="polite"
+                  aria-relevant="all"
+                  aria-atomic="true"
```

Comment:
We don't need these here AFAIK as they will be inherited from the parent fluent-tree-view